compiler/ir/backend.jvm/lower/src/org/jetbrains/kotlin/backend/jvm/JvmLower.kt

```diff
@@ -248,7 +248,7 @@ private val returnableBlocksPhase = makeIrFilePhase(
     ::ReturnableBlockLowering,
     name = "ReturnableBlock",
     description = "Replace returnable blocks with do-while(false) loops",
-    prerequisite = setOf(arrayConstructorPhase, assertionPhase)
+    prerequisite = setOf(arrayConstructorPhase, assertionPhase, directInvokeLowering)
```

Comment:
Note that this is a change from the previous commit. Since `DirectInvokeLowering` creates returnable blocks it should be a prerequisite of `ReturnableBlockLowering`.